packages/site/src/components/ContentCard.tsx

```diff
@@ -27,10 +34,23 @@ export const ContentCard = ({
       }}
     >
       {!component ? (
-        <img
-          style={{ width: "100%", mixBlendMode: "multiply" }}
-          src={imageURL}
-        />
+        <AtlantisThemeContextProvider dangerouslyOverrideTheme="light">
+          <div
+            style={
+              imageURL && theme === "dark"
+                ? {
```

Comment:
It's legit a practical example of how to use our stuff just right here in the code. Just wonderful.